aiohttp/multidict.py

```diff
@@ -305,6 +305,9 @@ def __init__(self, items):
     def __len__(self):
         return len(self._items)
 
+    def __repr__(self):
+        return '{0.__class__.__name__}({0._items!r})'.format(self)
```

Comment:
Maybe better to use `', '.join(repr(i) for i in self)` instead `repr(self._items)`?